app/src/main/java/ch/hikemate/app/utils/MapUtils.kt

```diff
@@ -264,4 +265,9 @@ object MapUtils {
         mapView.zoomLevelDouble,
         MapScreen.CENTER_MAP_ANIMATION_TIME)
   }
+
+  data class MapViewState(
+      val center: GeoPoint? = MapInitialValues().mapInitialCenter,
```

Comment:
Question: Is there a reason to use `GeoPoint` instead of `LatLng` here ?

Reply:
I did it since `MapInitialValues` uses GeoPoint